odl/contrib/fom/supervised.py

```diff
@@ -396,71 +437,60 @@ def blurring(data, ground_truth, mask=None, normalized=False,
     return mean_squared_error(data, ground_truth, mask, normalized)
 
 
-def false_structures(data, ground_truth, mask=None, normalized=False,
-                     smoothness_factor=None):
-    """Return weighted L2 distance, de-emphasizing regions defined by ``mask``.
-
-    .. note::
-        If the mask argument is omitted, this FOM is equivalent to the
-        mean squared error.
+def false_structures_mask(foreground, smoothness_factor=None):
```

Comment:
IMO this name makes about 0 sense to what the function does. Either emphasize it in the doc, or rename.

I mean, what is a false structure?

Reply:
`foreground_mask`?